activity/src/test/java/com/amazonaws/stepfunctions/cloudformation/activity/DeleteHandlerTest.java

```diff
@@ -38,20 +38,29 @@ public void setup() {
     }
 
     @Test
-    public void testSuccess() {
-        Mockito.when(proxy.injectCredentialsAndInvoke(Mockito.any(DescribeActivityRequest.class), Mockito.any(Function.class))).thenReturn(new DescribeActivityResult());
+    public void testDeleteHandler_pausesFor60sToStabilize_returnsSuccess() {
```

Comment:
Do we not also need a test case for when the stabilization time has successfully elapsed?

Reply:
The handler itself doesn't have context into how much time has elapsed. This is controlled in CloudFormation's workflow based on the ProgressEvent's `status` and `callbackDelaySeconds` that we return.

The additional work we're adding here is returning an `IN_PROGRESS` status with 60s delay, and then a Lambda in CloudFormation's workflow will re-invoke the Delete handler with the callbackContext we returned. On the second invocation, we return `SUCCESS` immediately.

We could theoretically mock a 60s wait in our unit test, but this is something completely outside our control, so I don't see a need for it.